src/agoradatatools/great_expectations/gx/json_schemas/immunohisto/points.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "http://example.com/example.json",
+    "type": "array",
+    "default": [],
+    "title": "Points Schema",
+    "items": {
+        "type": "object",
+        "title": "A Schema",
+        "required": [
+            "genotype",
+            "measurement",
+            "sex"
+        ],
+        "properties": {
+            "genotype": {
+                "type": "string"
+            },
+            "measurement": {
+                "type": [
+                    "number",
+                    "integer"
+                ]
+            },
+            "sex": {
+                "type": "string"
```

Comment:
This is definitely one where we could put a finite list of acceptable values.

Reply:
Should the only acceptable values be "male" and "female" or will we allow "unknown"?

Reply:
male and female only...if we don't know the sex of an animal, we can't use its data